caluma/caluma_workflow/domain_logic.py

```diff
@@ -212,6 +230,13 @@ def pre_skip(work_item, validated_data, user, context=None):
 
         validated_data["status"] = models.WorkItem.STATUS_SKIPPED
 
+        send_event(
+            events.pre_skip_work_item,
+            sender="pre_skip_work_item",
+            work_item=work_item,
+            user=user,
+            context=context,
+        )
```

Comment:
This should be done in `CompleteWorkItemLogic.pre_complete` by passing the event (like in `post_complete`)